lib/sqs/index.js

```diff
@@ -11,8 +11,9 @@ const compress = util.promisify(zlib.gzip);
 const decompress = util.promisify(zlib.gunzip);
 const { computeMessageSize, promisify, setHttpAgent } = require('../utils');
 
-// 256kb => b: (256kb * 1024 bytes =  262144)
-const MAX_MSG_SIZE = 262144;
+// Ideally MAX_MSG_SIZE should be 256kb => b: (256kb * 1024 bytes =  262144)
+// reducing MAX_MSG_SIZE due to SQS size calculation (might be different from the size calculation we do)
+const MAX_MSG_SIZE = 262085;
```

Comment:
during testing, I found that message size of 262101 bytes is allowed by SQS
adding some buffer of 12 bytes to the threshold here

Reply:
16 sounds good, we might have to inform other teams too about this change.

Reply:
we are the only team that utilizes this functionality, so we are good there.